web/src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -41,7 +75,7 @@
             </template>
           </v-card-title>
           <v-progress-linear
-            v-if="loading"
+            v-if="$asyncComputed.items.updating"
```

Comment:
As a matter of style, could this just be a check of `items === null`? Since the default value for that asyncComputed variable is `null`?

Reply:
I think this is actually clearer: it says that the progress bar should show precisely when the `items` value is updating, which is the intended semantics.

What is the style objection to doing it this way?

Reply:
I guess that you're relying on an implicitly injected Vue property :man_shrugging:. Honestly I don't feel to strongly about this, do whatever you feel is best.

Reply:
Regardless of style, if the update fails somehow and leaves `items` as `null`, using `items === null` will spin forever, which I think is worse than showing an empty list.

Reply:
> I guess that you're relying on an implicitly injected Vue property . Honestly I don't feel to strongly about this, do whatever you feel is best.

Since it's coming from a plugin we're actively using, I don't have any problem making use of the `$asyncComputed` injected property--it's about the same to me as making use of `$props` or anything else Vuey. @dchiquito's argument is strong as well, so let's keep this as it is.